recipe/meta.yaml

```
@@ -1,52 +1,76 @@
-{% set name = "FLAML" %}
-{% set version = "2.1.2" %}
+{% set name = "flaml" %}
+{% set version = "1.2.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 33102862a21c63d004f78e1349ccd55753e3534bfcdcb6e65a08562c14d69e44
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/FLAML-{{ version }}.tar.gz
+  sha256: ba8b62df4f459e226f951e03c4238a699b3168f708e30c4e7c75eba261896fe2
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # numpy 1.26.4(latest version <2.0) not avalible from py<39
+  # pandas 1.5.3(latest version <2.0) not avaliable for py>311
+  skip: True  # [py<39 or py>311]
+  skip: True  # [s390x]
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - wheel
+    - setuptools
   run:
-    - numpy >=1.17.0rc1
+    - python
+    # AttributeError: `np.Inf` was removed in the NumPy 2.0 release. Use `np.inf` instead.
+    # Set last compatible version for NumPy
+    - numpy >=1.17.0rc1,<=2.0
```

```suggestion
    - numpy >=1.17.0,<=2.0
```
Because we are not building release candidates, alpha, and beta versions of packages

---

done!
the patch is created.